app/services/registration_service.rb

```diff
@@ -11,9 +11,10 @@ def self.register(model:, workflow:, tags:)
     response = Dor::Services::Client.objects.register(params: model)
     druid = response.externalIdentifier
 
+    Dor::Services::Client.object(druid).administrative_tags.create(tags:) unless tags.empty?
+
     WorkflowClientFactory.build.create_workflow_by_name(druid, workflow, version: "1")
```

Comment:
AFAIK, creating admin tags before the creation of a workflow should be safe. Thoughts, @sul-dlss/infrastructure-team ?